docs/src/main/sphinx/release/release-391.md

```diff
@@ -0,0 +1,42 @@
+# Release 391 (20 July 2022)
+
+## General
+
+* Improve performance of repeated aggregations with ``CASE`` expressions. ({issue}`12548`)
+* Improve query performance when calling ``setBytes`` within a
+  ``synchronization`` section. ({issue}`13213`)
+* Fail queries that get stuck in long-running regular expression functions. ({issue}`12392`)
+* Set the default value of the ``optimizer.force-single-node-output``
+  configuration property to false. ({issue}`13217`)
+
+## BigQuery connector
+
+* Add support for reading external tables. ({issue}`13164`)
+* Add support for specifying a table comment and column comments when creating a
+  table. ({issue}`13105`)
+
+# Delta Lake connector
+
+* Fix incorrect results when using the Glue metastore with queries that contain
+  `IS NULL` and additional filters. ({issue}`13122`)
+
+## Hive connector
+
+* Add support for AWS Athena partition projection. ({issue}`11305`)
+* Fix creating empty ORC bucket files when using ZSTD. ({issue}`9775`)
```

Comment:
What's the fix? What was broken? How does it impact users (was it a failure, corrupted table, etc)?